Libraries/LibJS/Parser.cpp

```diff
@@ -269,6 +269,9 @@ NonnullRefPtr<Statement> Parser::parse_statement()
         consume();
         return create_ast_node<EmptyStatement>();
     default:
+        auto result = try_parse_labelled_statement();
```

Comment:
This should be wrapped in `if (match(TokenType::Identifier))` so we do not have to invoke the whole state pushing mechanism for every expression.

Reply:
Ideally we'd use token lookahead to match `Identifier` `Colon` directly, but oh well. Maybe in a next iteration of the parser.